packages/api/src/beacon/routes/debug.ts

```diff
@@ -196,5 +203,15 @@ export function getDefinitions(_config: ChainForkConfig): RouteDefinitions<Endpo
         timeoutMs: 5 * 60 * 1000,
       },
     },
+    getHistoricalSummaries: {
+      url: "/eth/v1/debug/historical_summaries",
+      method: "GET",
+      req: EmptyRequestCodec,
+      resp: {
+        data: ssz.capella.HistoricalSummaries,
+        meta: EmptyMetaCodec,
+        onlySupport: WireFormat.ssz,
```

Comment:
any reason why this shouldn't also support JSON?

Reply:
No specific reason other than it's most useful for portal clients to download the ssz encoded variant as it requires less work to integrate it into our own work.  Will add a JSON option as well.

Reply:
Generally we support both return formats and the clients decides based on Accept header, if header is not present we default to JSON

Reply:
This has been updated in the latest commit to support both.